src/common/OpQueue.h

```diff
@@ -66,6 +66,9 @@ class OpQueue {
   // Human readable brief description of queue and relevant parameters
   virtual void print(std::ostream &f) const = 0;
 
+  // Get the name pertaining to the type of OpQueue implementation
+  virtual std::string_view get_type() const = 0;
```

Comment:
Doing this as a string has at least two disadvantages.  First, and less important, is that we must do actual string comparisons which are a bit expensive.  Second, and more important, is that the compiler won't check that the value to which we are comparing is actually valid.  It would be very easy to mistype "mClockPriorityQueue" as "mClockPriortyQueue" and I doubt it would be caught in review.  Let's use an enum class instead.

Reply:
As suggested, I have introduced op_queue_type_t enum class in osd_types.h and made appropriate changes to OpQueue.h and OpScheduler.h to use it. PTAL.